### ISSUE TITLE
Package maintenance

### ISSUE THREAD

**Issue by tartieret** [escaped/django-inline-actions#62]
Hi @escaped,

I am using this nice package in a project and am starting to work on the transition to Django > 4
It looks like a few changes are required to make `django-inline-actions` compatible. Is there any way my team and I can help you make these changes over the next few weeks? We'll be happy to invest some time, but our contributions will have to be reviewed.

Please let me know what you think.

Thanks!

**Comment by jpmckinney** [comment]
I found https://github.com/HiroshiFuu/django4-inline-actions and https://github.com/pulilab/django5-inline-actions

**Comment by karlokr-p** [comment]
Hi there,

I want to thank the original authors of this code for providing this excellent package, it supports an important use case within our application. Unfortunately, we have noticed that this repository no longer seems to be maintained and has not received updates for Django 4/5 support.

We have decided to support the project by continuing maintenance via the fork https://github.com/panevo/django-admin-inlines. So far, we have patched the project to provide Django 4.x support (thanks to the contributions from HirushiFuu and pulilab) and updated all the tests to run against Django 4.x. The package is available for installation via pip https://pypi.org/project/django-admin-inlines/

We welcome any contributors to please submit issues and PR's to the new repository. Thanks!

**Comment by DmytroLitvinov** [comment]
Hi @karlokr-p ,
Could you please open possibility to open issues at your repo https://github.com/panevo/django-admin-inlines. Tried to upgrade my project to Django@5 and I see there is a strict requirement on Django `>=3.0,<5.0`

**Comment by tartieret** [comment]
> Hi @karlokr-p , Could you please open possibility to open issues at your repo https://github.com/panevo/django-admin-inlines. Tried to upgrade my project to Django@5 and I see there is a strict requirement on Django `>=3.0,<5.0`

@DmytroLitvinov I have just enabled issues on our fork  https://github.com/panevo/django-admin-inlines

**Comment by escaped** [comment]
Thanks for the offer to help with the Django >4 transition — and thanks to everyone who listed forks/workarounds here in the meantime.

Update: Django 5.2/6.x support landed via #64 and is included in 3.0.0, along with the recent inline-action fixes (#66–#69, #72–#73). If you're still on a fork, 3.0.0 from PyPI should be the path back to upstream.

Closing as addressed in 3.0.0 — please open a new issue for any remaining Django 5/6 problems.

*Triage assisted by Opencode (Muse Spark) — open vs. merged mapping verified before posting.*

**Comment by tartieret** [comment]
@escaped I am wondering if we should close our fork at https://github.com/panevo/django-admin-inlines or signal that people can go back to the original package. Do you need help maintaining this one? We are happy to commit some resources to help instead of managing a fork in parallel
